exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -194,22 +204,24 @@ func (b *BTCMarkets) Run() {
 
 	err := b.UpdateTradablePairs(forceUpdate)
 	if err != nil {
-		log.Errorf(log.ExchangeSys, "%s failed to update tradable pairs. Err: %s", b.Name, err)
+		log.Errorf(log.ExchangeSys,
+			"%s failed to update tradable pairs. Err: %s",
+			b.Name,
+			err)
 	}
 }
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
-func (b *BTCMarkets) FetchTradablePairs(asset asset.Item) ([]string, error) {
+func (b *BTCMarkets) FetchTradablePairs(a asset.Item) ([]string, error) {
```

Comment:
the param field isn't being used, could check asset.item if not SPOT and return an error.